emulators/kegworks/Portfile

```diff
@@ -19,17 +20,12 @@ homepage            https://github.com/Kegworks-App/Kegworks/
 description         ${name} is a user-friendly tool used to make ports of Microsoft Windows software to Apple's macOS.
 long_description    {*}${description}
 
-distname            winery-v${version}
-use_xz              yes
+livecheck.type      none
 
-checksums           rmd160  b9dfd427ae09246a12f28a780c538bfc3f94a109 \
-                    sha256  626f9bf9e6c29493906e371a6f1eb72acecb21cef68516890eb98ef59e844160 \
-                    size    817976
+pre-configure {
+    ui_error "Please do not install this port since it has been replaced by 'Sikarugir'."
+    return -code error
+}
 
-use_configure       no
-build {}
+distfiles
```

Comment:
remove, the `obsolete` PortGroup will show a message already and clears the `distfiles`.